app/javascript/elm/src/Data/SelectedSession.elm

```diff
@@ -169,3 +170,13 @@ view session heatMapThresholds linkIcon toMsg =
             , button [ class "button button--primary action-button action-button--copy-link", Events.onClick <| toMsg tooltipId, id tooltipId ] [ img [ src linkIcon, alt "Link icon" ] [] ]
             ]
         ]
+
+
+isSessionSelected : WebData SelectedSession -> Bool
```

Comment:
I guess you could use `https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/RemoteData#isSuccess`